### PR TITLE
python-flask-sqlalchemy: New package

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

File: mingw-w64-python-flask-sqlalchemy/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+             "${MINGW_PACKAGE_PREFIX}-python-pallets-sphinx-themes"
+             "${MINGW_PACKAGE_PREFIX}-python-sphinx-issues")
+checkdepends=("${MINGW_PACKAGE_PREFIX}-python-pytest"
+              "${MINGW_PACKAGE_PREFIX}-python-blinker"
+              "${MINGW_PACKAGE_PREFIX}-python-mock")
+source=("${_realname}-$pkgver.tar.gz::${url}/archive/${pkgver}.tar.gz")
+sha256sums=('b82dfd776a830c2fe75d9766f664f0e4a567523e70005cf57cfd3cfa10ce9d4f')
+
+prepare() {
+  rm -rf "python-build-${MSYSTEM}" | true
```
Comment (Biswa96) [inline mingw-w64-python-flask-sqlalchemy/PKGBUILD:31]: There `rm -rf` commands are not required.

Comment (podsvirov) [reply mingw-w64-python-flask-sqlalchemy/PKGBUILD:31]: Without precleaning, I'm having trouble rebuilding the package.

Comment (Biswa96) [reply mingw-w64-python-flask-sqlalchemy/PKGBUILD:31]: The `--cleanbuild` option with makepkg-mingw does that already.

Comment (podsvirov) [reply mingw-w64-python-flask-sqlalchemy/PKGBUILD]: Ok. I have removed these lines.

**Pull request merged**